### PR TITLE
Add steps for publishing a new Release

### PR DESCRIPTION
**Pull request by stevejalim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,3 +199,14 @@ Signal kwargs:
 - `instance`: The `Job` instance for which translation are being imported
 - `translation`: The `wagtail_localize.models.Translation` instance the translations are being imported to.
   Use `translation.get_target_instance()` to get the model instance that the translation is for (e.g. a page or snippet)
+
+## Cutting a new release
+
+1. Bump the version in <https://github.com/mozilla/wagtail-localize-smartling/blob/main/src/wagtail_localize_smartling/__init__.py>
+2. Update CHANGELOG
+3. Commit and land the changes in main (via a PR, or committing to main if you're _sure_ this won't cause clashes)
+4. Tag the release as `vX.Y.Z` on main. Remember to push up the new tag if you made it locally (`git push --tags`)
```
Comment (zerolab) [inline README.md:208]: fwiw, you can create the new tag directly from the new release page, but this works well too

Screenshot 2024-07-08 at 11 49 22

File: README.md [hunk 2]
```diff
@@ -203,3 +199,14 @@ Signal kwargs:
 - `instance`: The `Job` instance for which translation are being imported
 - `translation`: The `wagtail_localize.models.Translation` instance the translations are being imported to.
   Use `translation.get_target_instance()` to get the model instance that the translation is for (e.g. a page or snippet)
+
+## Cutting a new release
+
+1. Bump the version in <https://github.com/mozilla/wagtail-localize-smartling/blob/main/src/wagtail_localize_smartling/__init__.py>
+2. Update CHANGELOG
```
Comment (zerolab) [inline README.md:206]: suggestion

```suggestion
2. Update `CHANGELOG.md`
```

**Review by zerolab — CHANGES_REQUESTED** [review]
A small suggestion, and a note.

Worthy addition, thank you

File: README.md (hunk 1 above)
Comment (stevejalim) [reply README.md:208]: Good point! I've go so much into the habit of doing it locally that I forgot where to find it! Most of the team here tag locally so covering both bases is good. Will amend.

**Review by zerolab — APPROVED** [review]
:shipit:

**Pull request merged**